native/packages/react-native-bpk-component-text/src/__snapshots__/BpkText-test.android.js.snap

```diff
@@ -40,7 +40,6 @@ exports[`Android BpkText should render correctly with \`emphasize\` prop 1`] = `
       },
       Object {
         "fontFamily": "sans-serif-medium",
-        "fontWeight": "500",
```

Comment:
So, if this is removed and not replaced, is it just using the default font weight rather than explicitly setting something?

Reply:
it's using `sans-serif-medium` as font family which is bold

Reply:
Ah ok, cool!